src/neo/Network/P2P/Payloads/TransactionAttribute.cs

```diff
@@ -17,7 +17,7 @@ void ISerializable.Deserialize(BinaryReader reader)
             Usage = (TransactionAttributeUsage)reader.ReadByte();
             if (!Enum.IsDefined(typeof(TransactionAttributeUsage), Usage))
                 throw new FormatException();
-            Data = reader.ReadVarBytes(252);
+            Data = reader.ReadVarBytes(ushort.MaxValue);
```

Comment:
I think that it's very big, any use case for this size?

```suggestion
            Data = reader.ReadVarBytes(byte.MaxValue);
```

Reply:
I think 255 bytes may not be enough for remarks, also to keep consistent with the neo2.

Reply:
64kb for one attribute seems very huge